demos/local/simple_rag/setup_rag_agent.py

```
@@ -0,0 +1,630 @@
+        else:
+            print(f"❌ Vector database '{vector_db_id}' not found!")
+            raise ValueError(f"Vector database '{vector_db_id}' not found in registered databases")
+    except Exception as e:
+        print(f"⚠️  Could not list vector databases: {e}")
+    
+    # Debug: Check if RAG tool is available
+    try:
+        # Check what tools are available (this might fail on some llama-stack versions)
+        print(f"🔍 Checking RAG tool availability...")
+    except:
+        pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)